var/spack/repos/builtin/packages/libmbim/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Libmbim(AutotoolsPackage):
+    """Libmbim is a glib-based library for talking to WWAN modems and
+    devices which speak the Mobile Interface Broadband Model (MBIM)
+    protocol."""
+
+    homepage = "https://github.com/linux-mobile-broadband/libmbim/"
+    url      = "https://github.com/linux-mobile-broadband/libmbim/archive/1.18.0.tar.gz"
+
+    version('1.18.0', sha256='47003bfdf78bf32009a1d917f30c063079fa5bd4afc739d6d8ec356070b270df')
+    version('1.16.2', sha256='06b7a9e8430c6ab213d96c71a71469aefc86deb52cffd5e4f75121d9a79545e2')
+    version('1.16.0', sha256='d123426678f415c2ac4544534ed8a9ff54d133c2ba8c982ce667b793e54f8e99')
+    version('1.14.4', sha256='4b2e8723ea50b2e1d22695850c40abb9f7bcb713ea3b9f91f2c350aaa6ae8d1c')
+    version('1.14.2', sha256='bf161c4f78327f8422fd6a820e7e5571d99b719af45429e581bfd6a1585fe4a8')
```

Comment:
While you are at it, can you also add recent versions?

Reply:
OK, these packages were written in last year but not submitted, so maybe they are out of date.

Reply:
I tried latest version as `1.24.6` , `1.24.4` and `1.20.0`, and only `1.20.0` can build successfully.